docs/general/configuration_options.md

```diff
@@ -5,6 +5,7 @@ The following configuration options can be set on `ActiveModel::Serializer.confi
 ## General
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
+- `automatic_lookup`: Whether serializer should be automatically looked up or manually provided. Default: `true`
```

Comment:
@trek re:

> render json: photos, use_serializer: true

In https://github.com/rails/rails/pull/21496/ I have proposed [removing `use_renderers` since it appears never to have been used or tested](https://github.com/rails/rails/commit/556344639beeed059c1e8c0c27df18c6ef2efdc8).  Since your suggestion here has a similar name, I was wondering if it might be related to the `use_renderers` method in the Rails Render, or just coincidental